_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,207 @@
+- Even more terminal improvements (**@jdebp**, https://github.com/neovim/neovim/pull/6816)
+- The command line can now be colored (**@ZyX-l**, https://github.com/neovim/neovim/pull/6364)
+- You can now configure custom clipboard providers (**@nhooyr**, https://github.com/neovim/neovim/pull/6030)
+- For Windows users:
+  - Neovim can now be used on the console (**@equalsraf**, https://github.com/neovim/neovim/pull/6315)
+  - `:terminal` has been implemented (**@erw7**, https://github.com/neovim/neovim/pull/7007)
+  -  Path handling on windows has been improved (**@aignas**, https://github.com/neovim/neovim/pull/7349)
+- For Linux users:
+  - You can now download AppImage packages (**@AdnoC**, https://github.com/neovim/neovim/pull/6638)
+  - Function keys now work in the terminal UI (**@rjmill**, https://github.com/neovim/neovim/pull/5014)
+  - Very long lines do not crash Neovim anymore (**@Grimy**, https://github.com/neovim/neovim/pull/3527)
+  - `cquit` now takes an optional error code as argument (**@joshleeb**, https://github.com/neovim/neovim/pull/7336)
+  - A newly created `:terminal` will now stay in normal mode (**@nelstrom**, https://github.com/neovim/neovim/pull/6808)
+  - True colors in tmux have been fixed (**@DarkDefender**, https://github.com/neovim/neovim/pull/7100)
+  - Further improvements for vimtutor (**@fmoralesc**, https://github.com/neovim/neovim/pull/7028)
+  - Adjust terminal buffers when using the number column (**@tecywiz121**, https://github.com/neovim/neovim/pull/7440)
+
+
+#### 0.2.1-dev for Developers
+- Neovim sources are now analyzed with PVS (**@ZyX-l**, https://github.com/neovim/neovim/pull/6493)
+- External UIs can now draw the tabline (**@dzhou121**, https://github.com/neovim/neovim/pull/6583)
+- First usage of lua to implement core functionality (**@ZyX-l**, https://github.com/neovim/neovim/pull/6789)
+- Call lua directly from the API (**@bfredl**, https://github.com/neovim/neovim/pull/6704)
+- Programmatically get information about mappings via  `get_keymap` (**@tjdevries**, https://github.com/neovim/neovim/pull/6236)
+- `menu_get` now pretty-prints special chars, making it possible to feed its results back into e.g. `nvim_input` (**@teto**, **@KillTheMule**, https://github.com/neovim/neovim/pull/7340)
+ - You can now retrieve highlights `using nvim_get_hl_by_name/by_id` (**@teto**, https://github.com/neovim/neovim/pull/7082)
+ - `bufhl` can now be used to create new highlighting groups (**@bfredl**, https://github.com/neovim/neovim/pull/7414)
+ - GUIs now should use the `FocusGained` event instead of sending the `<FocusGained>` pseudokey (**@justinmk**, https://github.com/neovim/neovim/pull/7221)
+ - UI's can get menus from  `menu_get` (**@teto**, https://github.com/neovim/neovim/pull/6322)
+ - External UI's can now draw the command line (**@dzhou121**, **@bfredl**, https://github.com/neovim/neovim/pull/6162)
```

Comment:
maybe group with external tabline.

Reply:
or even all UI stuff, but even then the two new externalized features belong together.

Reply:
Yeah, you're right. I unified the wording, I'll check out a bit of grouping tomorrow.